docs/capturing-column-names.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Capturing Column Names in Mapped Data
+permalink: /capturing-column-names/
+---
+
+Column names themselves may contain data that should be included in each record. For example VCF files have a column name that is a Lab Number and it should be included on all records.
```

Comment:
just intrigued at the data structure here. I guess there a certain quirks that force this value to be at record level, rather than batch level?

Reply:
VCF is an unusual file format. It's one file per test which can contain many results/variants. Instead of having a data item for the lab number, it's the column header for one of the results columns.

So I'd expect to get a .zip file containing many VCFs, each of which will need to capture the lab number and assign it to any records created from that file.